docs/specification.md

```diff
@@ -89,12 +89,18 @@ its result.
 
 Types
 ```
+<<<<<<< HEAD
 tokenId = nat
 
 transferParam
   ( address :from_
   , address :to_
   , tokenId :tokenId
+=======
+transferParam
+  ( address :from_
+  , address :to_
+>>>>>>> Fix specification
```

Comment:
:fffuuu:

Reply:
Thank you github, very cool

Reply:
Ok, I guess it's up to you to complete this MR

Reply:
Ok, let me deal with the specification.